tasks/jasmine.js

```diff
@@ -194,7 +194,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.display === 'full' || options.showLogging) {
```

Comment:
Since explicitly setting `showLogging = false` should not be overridden by `display = full`, it might be more intuitive to have `showLogging` default to `true` and have this logging not depend on `display`.

```js
if (options.showLogging) {
```

|   | display = full (default) | display = short |
|---|---|---|
| **showLogging = true (default)** | log | log |
| **showLogging = false** | silent | silent |

To preserve backwards compatibility with `display = short`, `showLogging` would need to default to a 3rd value like `undefined`, which exhibit the old "undefined" logging behavior. ([See earlier comment](https://github.com/gruntjs/grunt-contrib-jasmine/pull/236#discussion_r67083405))